software/partdb.yml

```diff
@@ -0,0 +1,14 @@
+name: "Part-DB"
+website_url: "https://docs.part-db.de/"
+source_code_url: "https://github.com/Part-DB/Part-DB-server"
+description: "Part-DB is an Open-Source inventory management system for your electronic components."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Shell
+  - Docker
+  - Nodejs
```

Comment:
From [their Documentation](https://docs.part-db.de/installation/installation_guide-debian.html), no shell is needed. However, PHP is.

```suggestion
  - PHP
  - Nodejs
  - Docker
```